front/src/applications/operationalStudies/views/v2/ScenarioV2.tsx

```diff
@@ -96,12 +96,13 @@ const ScenarioV2 = () => {
     if (isScenarioError && errorScenario) throw errorScenario;
   }, [isScenarioError, errorScenario]);
 
-  const { updateInfraID, updateTimetableID } = useOsrdConfActions();
+  const { updateInfraID, updateTimetableID, updateElectricalProfileSetID } = useOsrdConfActions();
 
   useEffect(() => {
     if (scenario) {
       dispatch(updateTimetableID(scenario.timetable_id));
       dispatch(updateInfraID(scenario.infra_id));
+      dispatch(updateElectricalProfileSetID(scenario.electrical_profile_set_id));
```

Comment:
- use Id instead of ID in the name of the action
- add a small test to dispatch undefined rather than null (I'm not sure this suggestion works, but you can try :) otherwise, a classic ternary test with `... ? ... : ...` will do the job)
```suggestion
      dispatch(updateElectricalProfileSetId(scenario.electrical_profile_set_id || undefined));
```

Reply:
Null has been removed so it's all good